helm/cluster-vsphere/values.yaml

```diff
@@ -1,134 +1,268 @@
+      #   # Metric is the weight/priority of the route.
+      #   # <integer> -required-
+      # - metric: 50
+      #   # To is an IPv4 or IPv6 address.
+      #   # <string> -required-
+      #   to: ""
+      #   # Via is an IPv4 or IPv6 address.
+      #   # <string> -required-
+      #   via: ""
+
+
+    # NumCPUs is the number of virtual processors in a virtual machine. Defaults
+    # to the eponymous property value in the template from which the virtual
+    # machine is cloned.
+    # <integer>
+    # numCPUs: 0
+
+    # NumCPUs is the number of cores among which to distribute CPUs in this
+    # virtual machine. Defaults to the eponymous property value in the template
+    # from which the virtual machine is cloned.
+    # <integer>
+    # numCoresPerSocket: 0
+
+    # OS is the Operating System of the virtual machine Defaults to Linux
+    # <string>
+    # os: "Linux"
+
+    # PciDevices is the list of pci devices used by the virtual machine.
+    # pciDevices: []
+      # # DeviceID is the device ID of a virtual machine's PCI, in integer. Defaults
+      # # to the eponymous property value in the template from which the virtual
+      # # machine is cloned.
+      # # <integer>
+      # deviceId: 0
+      # # VendorId is the vendor ID of a virtual machine's PCI, in integer. Defaults
+      # # to the eponymous property value in the template from which the virtual
+      # # machine is cloned.
+      # # <integer>
+      # vendorId: 0
+
+    # ProviderID is the virtual machine's BIOS UUID formated as
+    # vsphere://12345678-1234-1234-1234-123456789abc
+    # <string>
+    # providerID: ""
+
+    # ResourcePool is the name or inventory path of the resource pool in which
+    # the virtual machine is created/located.
+    # <string>
+    # resourcePool: ""
+
+    # Server is the IP address or FQDN of the vSphere server on which the virtual
+    # machine is created/located.
+    # <string>
+    # server: ""
+
+    # Snapshot is the name of the snapshot from which to create a linked clone.
+    # This field is ignored if LinkedClone is not enabled. Defaults to the
+    # source's current snapshot.
+    # <string>
+    # snapshot: ""
+
+    # StoragePolicyName of the storage policy to use with this Virtual Machine
+    # <string>
+    # storagePolicyName: ""
+
+    # TagIDs is an optional set of tags to add to an instance. Specified tagIDs
+    # must use URN-notation instead of display names.
+    # <[]string>
+    # tagIDs: []
+
+    # Template is the name or inventory path of the template used to clone the
+    # virtual machine.
+    # <string> -required-
+    template: ""
+
+    # Thumbprint is the colon-separated SHA-1 checksum of the given vCenter
+    # server's host certificate When this is set to empty, this VirtualMachine
+    # would be created without TLS certificate validation of the communication
+    # between Cluster API Provider vSphere and the VMware vCenter server.
+    # <string>
+    # thumbprint: ""
```

Comment:
Container registry configuration is also missing. See https://github.com/giantswarm/cluster-cloud-director/blob/main/helm/cluster-cloud-director/values.yaml#L120

Reply:
yea, I didn't have the latest version of `cluster-cloud-director` repo pulled when doing these changes (same as for the pub keys for ssh)

Reply:
done in commit https://github.com/giantswarm/cluster-vsphere/pull/23/commits/19db703f0804b951714f04d366e681bddc4f9573 (on this PR)